frontend/src/components/GroupView/index.tsx

```diff
@@ -1,15 +1,22 @@
 import React, { ReactElement } from 'react';
 import MiddleBar from './MiddleBar';
+import RightView from './RightView';
 import styles from './index.module.css';
 
 type Props = {
   groupName: string;
 }
 
+const members = ['Darien Lopez', 'Sarah Johnson', 'Michelle Parker', 'Samwise Bear'];
```

Comment:
There is a discrepancy between the number of people in Group Tasks view and the People section because I defined my `members` for People in `People/index.tsx`. I will push in a commit that makes the People section use your `members` array.